### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by morgz**
Update[ instructions](https://github.com/getsentry/sentry-elixir/pull/472#issuecomment-946672478) for handling crashed process exceptions

### CODE REVIEW

**Comment by morgz** [comment]
> @morgz - Because of the issue you logged, I want to confirm is this is the page you intend to update? I ask because the file you're updating isn't actually displayed in the docs, but is actually part of the project setup wizard in the application. It just lives in our docs repo. The closest thing I can find to this in the docs is this:
> https://docs.sentry.io/platforms/elixir/configuration/options/#capture-crashed-process-exceptions

ah ok - It could be that your docs are up-to-date and it's the [wizard / getting started step](https://sentry.io/func-main/elixir/getting-started/elixir/) which is outdated. As far as I understand it, when you refer to capturing crashed process exceptions you should use the latest example from Jose [here](https://github.com/getsentry/sentry-elixir/pull/472/commits/a31118e171ec61dc895e1e0a2c14be36665cc6ae) I've just done a PR for the page I found this outdated.

**Comment by imatwawana** [comment]
@AbhiPrasad is this ready to merge?

**Comment by AbhiPrasad** [comment]
Ah sorry - left this in progress 😅. We’ll need to fix the code snippet block. It doesn’t have a ``` at the end. Will do that and then merge.

File: src/wizard/elixir/index.md [hunk 1]
```diff
@@ -85,12 +85,15 @@ plug Sentry.PlugContext
...
-config :logger,
-  backends: [:console, Sentry.LoggerBackend]
-```
+This library comes with an extension to capture all error messages that the Plug handler might not.  This is based on [Logger.Backend](https://hexdocs.pm/logger/Logger.html#module-backends). You can add it as a backend when your application starts:
+
+```elixir
+# lib/my_app/application.ex
+
+def start(_type, _args) do
+  Logger.add_backend(Sentry.LoggerBackend)
```
Comment (AbhiPrasad) [inline src/wizard/elixir/index.md:96]:
```suggestion
This library comes with an extension to capture all error messages that the Plug handler might not.  This is based on [Logger.Backend](https://hexdocs.pm/logger/Logger.html#module-backends). You can add it as a backend when your application starts:

```elixir
# lib/my_app/application.ex

def start(_type, _args) do
  Logger.add_backend(Sentry.LoggerBackend)
```
```

**Comment by AbhiPrasad** [comment]
Created https://github.com/getsentry/sentry-docs/pull/4386 to make the necessary changes, so closing this PR in favour of that one.

**Pull request closed without merging**